doc/readme.md

```diff
@@ -82,12 +82,12 @@ Certain commands have options (`platformOpts`) that are specific to a particular
         cd myApp
```

Comment:
In readme.md... Can I remove: lines 213-214 AND line 429 AND 438? Or should I just change these to --nofetch instead of --fetch?